src/database/database-leveldb.cpp

```diff
@@ -97,5 +98,101 @@ void Database_LevelDB::listAllLoadableBlocks(std::vector<v3s16> &dst)
 	delete it;
 }
 
-#endif // USE_LEVELDB
+AuthDatabaseLevelDB::AuthDatabaseLevelDB(const std::string &savedir)
+{
+	leveldb::Options options;
+	options.create_if_missing = true;
+	leveldb::Status status = leveldb::DB::Open(options,
+		savedir + DIR_DELIM + "auth.db", &m_database);
+	ENSURE_STATUS_OK(status);
+	m_savedir = savedir;
```

Comment:
Looks like `m_savedir` isn't used anywhere.

Reply:
Thank you for pointing this out.